pilopt/src/qse_opt.rs

```diff
@@ -154,8 +154,6 @@ where
     assert!(exponent >= 0.into());
     if exponent == 0.into() {
         QuadraticSymbolicExpression::from(T::from(1))
-    } else if exponent == 1.into() {
-        v.clone()
```

Comment:
why?

Reply:
I believe that the `exponent == 1` case is handled just fine by the other cases already.

Reply:
It creates an unnecessary multiplication by 1 but ok...